apps/src/applab/gridUtils.js

```diff
@@ -25,8 +25,8 @@ export function scaledDropPoint(draggedElement) {
   const xScale = boundingRect.width / div.offsetWidth;
   const yScale = boundingRect.height / div.offsetHeight;
 
-  let left = (draggedOffset.left - boundingRect.left) / xScale;
-  let top = (draggedOffset.top - boundingRect.top) / yScale;
+  let left = (draggedOffset.left - $(div).offset().left) / xScale;
+  let top = (draggedOffset.top - $(div).offset().top) / yScale;
 
```

Comment:
Sounds reasonable. This seems like it might show up in eyes tests if there is any difference. Please notify the current/next dev of the day when you merge this so they know which PR to look at if there are problems.